docs/css/main.css

```diff
@@ -56,28 +56,28 @@ mark {
     margin: 0 auto;
     min-width: 0;
     max-width: 1000px;
+    overflow-x: auto;
     padding: 0.8rem 20px 0 20px;
     transition: 0.4s;
     -webkit-transition: 0.4s;
 }
 
 #site-nav,
 #page-nav {
+    display: flex;
+    flex-direction: column;
     position: sticky;
-    top: 0;
+    top: var(--sticky-header-height);
     flex: 0 0 auto;
-    padding-top: 1rem;
     max-width: 300px;
+    max-height: calc(100vh - var(--sticky-header-height));
```

Comment:
last change: `100vh` worked before as it took into account `padding-top` from `fixed-header-padding`.
With the shift to `top: var(--sticky-header-height);` the content boxes actually do shift down, so we need to minus that off.

The other changes (`90vh`) is just to standardise the way this is done in the layout stylesheet between the site and page nav to make things simpler.